contracts/LpSugar.vy

```diff
@@ -837,15 +867,24 @@ def _cl_lp(_data: address[4], _token0: address, _token1: address) -> Lp:
   staked1: uint256 = 0
   tick_spacing: int24 = pool.tickSpacing()
   pool_liquidity: uint128 = pool.liquidity()
-  gauge_liquidity: uint128 = pool.stakedLiquidity()
   token0_fees: uint256 = 0
   token1_fees: uint256 = 0
 
   slot: Slot = pool.slot0()
   tick_low: int24 = slot.tick - tick_spacing
   tick_high: int24 = slot.tick + tick_spacing
 
-  if gauge.address == empty(address) or gauge_liquidity == 0:
+  tickInfo: TickInfo = pool.ticks(slot.tick)
+  stakedLiquidityNet: int128 = tickInfo.stakedLiquidityNet
+
+  if stakedLiquidityNet == 0:
+    slnMinus: int128 = pool.ticks(slot.tick-1).stakedLiquidityNet
+    slnPlus:  int128 = pool.ticks(slot.tick+1).stakedLiquidityNet
+    stakedLiquidityNet = convert((convert(slnPlus, uint256) + convert(slnMinus, uint256) + convert(stakedLiquidityNet, uint256)) / 3, int128)
+
+  active_liquidity: uint128 = convert(abs(convert(stakedLiquidityNet, int256)), uint128)
```

Comment:
@pedrovalido I can still see we're using the immediate ticks liquidity. I don't think we should do that. Let's just use the current active tick `stakedLiquidityNet` for now.

Reply:
my bad 🤦‍♂️. apologies. changes done